dev/com.ibm.ws.javaee.ddmodel/resources/com/ibm/ws/javaee/internal/ddmodel/resources/DDModelMessages.nlsprops

```diff
@@ -381,3 +381,27 @@ incorrect.descriptor.namespace.for.version=CWWCK27789W: The {0} deployment descr
 incorrect.descriptor.namespace.for.version.explanation=A specific namespace is expected within the deployment descriptor for the specified version. The incorrect namespace was ignored.
 incorrect.descriptor.namespace.for.version.useraction=Change the namespace within the deployment descriptor to the expected value.
 
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} is the path to a deployment descriptor, including paths to parent archives
+#    {1} is the number of a line within the deployment descriptor
+#    {2} is the value of a 'name' attribute within a session configuration 'attribute' element
+#    {3} is the value of the 'value' attribute of a session configuration 'attribute' element
+#    {4} is the value of the 'value' attribute of a session configuration 'attribute' element
+# -----------------------------------------------------------------------
+duplicate.session.config.attribute.name=CWWCK27790W: The {0} deployment descriptor on line {1} duplicates the ''{2}'' ''name'' of a session configuration attribute elements.  The ''{3}'' ''value'' will be associated with the name instead of the previous ''{4}'' ''value''.
+duplicate.session.config.attribute.name.explanation=The deployment descriptor has multiple attribute elements which have the same name and which are within the same session configuration element. In Request-For-Comments (RFC) 6265, ''HTTP State Management Mechanism'', dated April 2011, the section 4.1.1, ''Syntax'', states ''To maximize compatibility with user agents, servers SHOULD NOT produce two attributes with the same name in the same set-cookie-string.''. Consequently, the descriptor parser ignores all but the last attribute element which have the same name.
```

Comment:
```suggestion
duplicate.session.config.attribute.name.explanation=The deployment descriptor has multiple attribute elements which have the same name and which are within the same session configuration element. In Request-For-Comments (RFC) 6265, ''HTTP State Management Mechanism'', dated April 2011, section 4.1.1, ''Syntax'', states ''To maximize compatibility with user agents, servers SHOULD NOT produce two attributes with the same name in the same set-cookie-string.''. Consequently, the descriptor parser ignores all but the last attribute element which have the same name.
```

 In Request-For-Comments (RFC) 6265, ''HTTP State Management Mechanism'', dated April 2011, section 4.1.1 - is there a need to specify this detail here?